prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -236,8 +247,30 @@ class LivdTableDownload : CliktCommand(
         // Write to CSV file.
         fileOutputStream.write(data.toString().toByteArray())
         fileOutputStream.close()
-
-        TermUi.echo("\tSUCCESS: The $outputfile file is created.\n")
         return true
     }
+
+    /**
+     * Updates the LIVD lookup table name [livdLookupTable] of CSV file.  It setups PRIME CLI Lookup Table Create
+     * Command line options.  And then, it calls the create lookup table command to create the new version of lookup
+     * table.  Note, it always create the new version regardless since it uses -f option.
+     */
+    private fun updateTheLivdLookupTable(livdLookupTable: File): Boolean {
+
+        // The environment the command needs to run on.
+        val environment = Environment.get(env)
+
+        TermUi.echo("Creating $livdSARSCov2File table ...")
+        val args: MutableList<String> = mutableListOf(
+            "-e", environment.toString().lowercase(), "-n", livdSARSCov2File,
+            "-i", livdLookupTable.absolutePath, "-s"
```

Comment:
After playing with it, the -s also needs to be removed, so we can get the diff and prompt if we want to create the new table version or not.

Reply:
Done.